modules/nw-udn-best-practices.adoc

```diff
@@ -19,7 +19,8 @@ Before setting up a `UserDefinedNetwork` (UDN) resource, users should consider t
 +
 [IMPORTANT]
 ====
-For {product-title} 4.17 and later, clusters use `169.254.0.0/17` for IPv4 and `fd69::/112` for IPv6 as the default masquerade subnet. These ranges should be avoided by users. For updated clusters, there is no change to the default masquerade subnet.
+* For {product-title} 4.17 and later, clusters use `169.254.0.0/17` for IPv4 and `fd69::/112` for IPv6 as the default masquerade subnet. These ranges should be avoided by users. For updated clusters, there is no change to the default masquerade subnet.
+* Changing the cluster's masquerade subnet cannot be done after a user-defined network has been configured for a project. Attempting to modify the masquerade subnet after a UDN has been set up can disrupt the network connectivity and cause configuration issues.
```

Comment:
it will actually cause stale VRFs and kernel routes. Is it better to just mention that it will cause connectivity and configuration issues or better to give info about what is actually gonna happen if anyone does change masquerade subnet at day 2?
cc: @tssurya

Reply:
@arghosh93 former is OK i.e network connectivity and cause config issues is fine, we don't need to tell specifics

"Is not supported" would be the word I use though... instead of cannot be done... because you know you can do it.. just that it won't work well for you -> hence the docs

Reply:
@arghosh93 @tssurya Thanks. Fixed!